api/v1/helpers/authUtils.js

```diff
@@ -40,6 +40,28 @@ function getUser(req) {
 } // getUser
 
 /**
+ * Determines whether the user has write access to a model
+ *
+ * @param {Request} req - The request object
+ * @param {String} modelName - The name of the model to write to
+ * @returns {Promise} - A promise which resolves to true if the user has
+ *  write access to the resource
+ */
+function hasWriteAccess(req, modelName) {
```

Comment:
Please rename this function to "profileHasWriteAccess". Otherwise people will confuse it with record-level write access.